test/com/stanimiriliev/javaobjects/sumator/SumatorTest.java

```diff
@@ -0,0 +1,37 @@
+package com.stanimiriliev.javaobjects.sumator;
```

Comment:
test classes should n placed in a separate package. tests should be separated from the production source code, cause they are use only for testing the code, they are not used in the production app. It is easier to filter them on the build process when preparing the app